### DISCUSSION TITLE
Uso da APyB para gerenciar senhas da Python Nordeste

### DISCUSSION THREAD

**Discussion by cecivieira** [apyb/comunidade#66] [Geral]
Olá,

É possível centralizar o chaveiro da Python Nordeste na APyB? Se sim, como faz?
Hoje as nossas senhas estão todas em um documento de texto plano no Google Drive.

cc @maribedran

**Comment by rougeth** [comment]
Hoje, a APyB não consegue ajudar com as senhas, mas é um tema recorrente e vamos levar esse caso em consideração.

**Comment by maribedran** [comment]
@rougeth acho que pras organizações locais a questão maior é o pagamento. A gente não precisa necessariamente que o cofre seja da APyB, mas precisa de um CC pra fazer um plano desses que permita mais de dois usuários. Como o valor é pequeno, acredito que possa até sair do caixa do evento local, mas o pagamento teria que ser via APyB.

**Reply by rougeth** [reply]
CC == cartão de crédito? hahaha Se sim, APyB não tem (ainda). Nós estamos analisando algumas alternativas para as nossas senhas, e temos um requisito semelhante, já que temos que compartilhar acessos as pessoas que estão organizando a Python Brasil. Acho que conseguimos chegar numa solução que contemple outros eventos também.

**Reply by maribedran** [reply]
Isso! hahahaha
Como seria um pagamento recorrente, imagino que todos esses serviços sejam pagos por cartão. Seria ruim ter alguém pagando individualmente e recebendo reembolso por exemplo porque se der problema todo mundo perde o acesso.

**Comment by cecivieira** [comment]
@maribedran se o dinheiro não sairá do caixa da APyB, então cabe a organização de cada evento incluir o custo nos gastos e solicitar pagamento, como é feito com qualquer outro item relacionado ao evento.
